### ISSUE TITLE
Support source mapping of stack traces in the Debug Console

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode-chrome-debug-core#6]
_From @auchenberg on March 22, 2016 22:11_

When an error is thrown with a stack trace, we should support source mapping of the source:

Actual:
![app js - timey in 2016-03-22 15-09-38](https://cloud.githubusercontent.com/assets/173559/13969430/1bf126c8-f040-11e5-88cf-8bd3f7cf5b48.png)

Click on `Scope.$scope.onAddNewClick (http://localhost:8080/app_bundled.js:234:5)` triggers:
![app js - timey in 2016-03-22 15-10-13](https://cloud.githubusercontent.com/assets/173559/13969441/2fe6e8ca-f040-11e5-841e-46b91258e494.png)

Expected: 
Path to files within stack traces are source mapped to the original source. 

_Copied from original issue: Microsoft/vscode-chrome-debug#132_

**Comment by nojvek** [comment]
@roblourens Seems like this issue has been open for a while. I can take a stab at it if you're busy with other tasks. Let me know.

**Comment by auchenberg** [comment]
@nojvek Please hack away, if time allows it :)

**Comment by roblourens** [comment]
That would be cool. I don't have any immediate plan to work on this.

**Comment by auchenberg** [comment]
Landed in https://github.com/Microsoft/vscode-chrome-debug-core/pull/190

**Comment by nojvek** [comment]
Just curious, how often do you release updates to the chrome debug plugin?

Also do you envision the plugin to be part of vscode install by default?

**Comment by auchenberg** [comment]
We release as needed, and for know we don't ship out of the box until we
have a better way to enable remote debugging in Chrome (and other browsers).

**Comment by roblourens** [comment]
I update it whenever I have time to test, and write a changelog and stuff. Planning on doing it by today or tomorrow.

Probably will stay a separate extension for the time being.

**Comment by nojvek** [comment]
Makes sense.

node2 inspect debugger ends up using the same core as chrome debugger
though right?

**Comment by roblourens** [comment]
Yep, also I just updated the chrome extension :)

**Comment by nojvek** [comment]
⛄️🎉🎊💫✨🎈💥🔥

**Comment by aramando** [comment]
Any ideas why this still isn't working for me? Do you have to have sourcemap files for the JS files in question?

**Comment by roblourens** [comment]
I just (like an hour ago) updated it to map from URL to local file, if you aren't using sourcemaps. If you are using sourcemaps, then it should work if setting BPs and debugging with sourcemaps works.

**Comment by aramando** [comment]
Oh, fantastic! So I just need to wait for that to make it into the next release of the extension? Any idea roughly what kind of timeframe that might occur in?

**Comment by roblourens** [comment]
The release went out today, it's in 2.7.3 if you don't already have that

**Comment by aramando** [comment]
Oh, I am already running that, but it's still not working :-/

**Comment by roblourens** [comment]
Can you open another issue with details?

**Comment by AskYous** [comment]
Hey @roblourens, I am also receiving an error saying it cannot find the file. I am running **Debugger for Chrome version 3.1.2**. I posted a screenshot below. Notice a couple of things:

1. My website is running on localhost:1337. For some reason, VSCode is trying to open the file in /1337/js/components.js. (How did the port get into the source path???)
2. I have source mapping on for my typescript files.
3. The file open is the launch.json file shown in the screenshot. I pointed the `webRoot` folder to my assets folder, yet it seems to completely ignore it.

![image](https://user-images.githubusercontent.com/791996/27094064-c34f472c-501d-11e7-998a-0b8691809e3a.png)

**Comment by roblourens** [comment]
That's strange, but a different problem, can you open a new issue?

Can you also set `"trace": true` in your launch config, repro this again, and include the log?

**Comment by AskYous** [comment]
@roblourens Sure, I'm on it. I got here through a google search. Question: where does the debug info log into when I turn trace on in Windows 10? I don't know where it is.

**Comment by roblourens** [comment]
It writes a log file and prints the path at the top of the console.

**Comment by AskYous** [comment]
Ok. I opened #212
